definitions/capabilities.js

```diff
@@ -3,56 +3,56 @@ const SonyBraviaAndroidTvCommunicator = require('../helpers/sony-bravia-android-
 module.exports = [
   {
     name: 'onoff',
-    function: (device, data, value) => {
-      console.log(`${data.name} setting device power state to: `, value ? 'ON' : 'OFF');
+    function: (device, _value) => {

```

Comment:
Could you explain the convention behind `_value` vs `value`?

Reply:
Just unified all to same format here.